server/context.go

```diff
@@ -141,6 +141,11 @@ func NewContext() *Context {
 	return ctx
 }
 
+// Insecure indicates whether or not SSL is to be used.
+func (ctx *Context) Insecure() bool {
```

Comment:
either that, or simply allow the `COCKROACH_INSECURE` env variable as well. Since people will just be following the docs, they'll just be doing copy/paste, so it seems easy enough.